onnx/defs/rnn/defs.cc

```diff
@@ -6,12 +6,20 @@
 using AttrType = onnx::OpSchema::AttrType;
 using namespace onnx;
 
+namespace onnx {
+
 std::function<void(OpSchema&)> RNNDocGenerator(const char* name) {
     return [=](OpSchema& schema) {
         schema.Attr("direction", "Specify if the RNN is forward, reverse, or bidirectional. "
                     "Must be one of forward (default), reverse, or bidirectional.",
                     AttrType::STRING);
         schema.Attr("hidden_size", "Number of neurons in the hidden layer", AttrType::INT);
+        schema.Attr("alpha", "Optional scaling values used by some activation functions.", AttrType::FLOATS);
```

Comment:
it's unclear how they are specified. I.e. if I have `activations='tanh,prelu'`, should `alpha` be `0,0.1` or just `0.1`

also should be called `activations_alpha` for clarity?

Reply:
It should just be `0.1`. I updated the doc to make it more clear hopefully.

Changed to `activation_alpha` and `activation_beta`.